src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -238,6 +246,13 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                             libraryTab.clearAndSelect(entry.localEntry());
                         });
                         vContainer.getChildren().add(jumpTo);
+
+                        Button compareButton = IconTheme.JabRefIcons.MERGE_ENTRIES.asButton();
+                        compareButton.setTooltip(new Tooltip(Localization.lang("Compare with duplicate entries")));
```

Comment:
I think "duplicate" is not a good term. It is about an existing entry in the library

```suggestion
                        compareButton.setTooltip(new Tooltip(Localization.lang("Compare with existing entries")));
```